customize.dist/src/less2/include/avatar.less

```diff
@@ -26,7 +26,6 @@
 & {
     .avatar_vars();
     &.cp-avatar {
-        overflow: hidden;
```

Comment:
What was the issue with `overflow: hidden`? I can't really see a use-case for having the avatar bigger than its container